src/app/Fake.Windows.Registry/Registry.fs

```diff
@@ -0,0 +1,153 @@
+﻿/// Contains functions which allow to read and write information from/to the registry.
+///
+/// ## Sample
+/// 
+/// #### Create a subkey
+///     let subkey = "Company/MyApp"
+///     Registry.createRegistrySubKey Registry.HKEYCurrentUser subkey
+///
+/// #### Write a key-value pair to a subkey
+///     Registry.setRegistryValue Registry.HKEYCurrentUser subkey "AppType" "Premium"
```

Comment:
Now I get it, I think an empty line is required above in order for it to be recognized as code. But I can fix that